cmd/agent/app/integrations.go

```diff
@@ -591,18 +591,18 @@ func validateRequirement(version *semver.Version, comp string, versionReq *semve
 	}
 }
 
-func minAllowedVersion(integration string) (*semver.Version, error) {
+func minAllowedVersion(integration string) (*semver.Version, bool, error) {
 	here, _ := executable.Folder()
 	lines, err := ioutil.ReadFile(filepath.Join(here, relReqAgentReleasePath))
 	if err != nil {
-		return nil, err
+		return nil, false, err
 	}
-	version, err := getVersionFromReqLine(integration, string(lines))
+	version, found, err := getVersionFromReqLine(integration, string(lines))
 	if err != nil {
-		return nil, err
+		return nil, false, err
 	}
 
-	return version, nil
+	return version, found, nil
 }
 
 // Return the version of an installed integration, or nil if the integration isn't installed
```

Comment:
If we could change this one as well while we're here. 
Going with the same pattern for `installedVersion` would be great as well. But it's not a blocker